website/src/pages/docs/features/apollo-federation.mdx

```diff
@@ -28,44 +28,190 @@ we can use GraphQL Yoga to build our gateway server.
 
 ## Gateway
 
-### Installation for Gateway
+You can use GraphQL Yoga to implement a Gateway exposing an Apollo Federation supergraph. For this,
+the gateway needs to have access to the supergraph SDL.
+
+You have the choice between loading it from a schema registry (like Hive or Apollo GraphOS) or
+locally from the file system.
+
+### Using Hive
+
+Hive is a schema registry and usage reporting plateform for GraphQL. You can use it to manage an
+Apollo Federation supergraph.
+
+For a full step by step setup getting started, please follow this
+[Apollo Federation usage guide](https://the-guild.dev/graphql/hive/docs/get-started/apollo-federation).
+
+#### Installation
 
 ```sh npm2yarn
-npm i @apollo/gateway @envelop/apollo-federation graphql-yoga graphql@15
+npm i graphql-yoga graphql @graphql-yoga/hive
 ```
 
-### Example Gateway
+#### Example
 
-```ts
-import { createServer } from 'http'
+```js filename="src/index.mjs"
+import { createServer } from 'node:http'
 import { createYoga } from 'graphql-yoga'
-import { ApolloGateway } from '@apollo/gateway'
-import { useApolloFederation } from '@envelop/apollo-federation'
-
-// Initialize the gateway
-const gateway = new ApolloGateway({
-  serviceList: [
-    { name: 'accounts', url: 'http://localhost:4001' },
-    { name: 'products', url: 'http://localhost:4002' }
-    // ...additional subgraphs...
+import { useHive } from '@graphql-hive/yoga'
+import { schema } from './schema'
+
+const yoga = createYoga({
+  schema,
+  plugins: [
+    useHive({
```

Comment:
Hmm unfortunately Hive plugin doesn't seem to set any schema or executor for supergraph. Is it really relevant here?

Reply:
Ah yeah I forgot that ><'

We should have a very simple way to use it exactly as for Apollo Managed Federation. Would be strange to have a more straightforward way to integrate with Apollo than with Hive

Reply:
I see but this plugin is a monitoring plugin which is not equivalent of Managed Federation one.
A future GraphOS monitoring plugin will be the equivalent.

Reply:
Yes, we probably need a dedicated plugin for that.

Until we have it, I remove the Hive section ?

Reply:
Let's not mention Hive until we have a clear recipe. We'll cover a complicated scenario in Mesh already with polling retry etc.

Reply:
I've replaced the example with how to use Hive's supergraph sdl fetcher combined with stitching.